src/linpred.jl

```diff
@@ -272,7 +306,46 @@ end
 inverse(x::DensePred) = invchol(x)
 inverse(x::DensePredQR) = invqr(x)
 
-vcov(x::LinPredModel) = rmul!(inverse(x.pp), dispersion(x, true))
+working_residuals(x::LinPredModel) = x.rr.wrkresid
```

Comment:
Consistent with other methods, call this on the `rr` field and define a method for it:
```suggestion
working_residuals(x::LinPredModel) = working_residuals(x.rr)
```